<ui version="4.0" >
 <class>KLADRTreePopup</class>
 <widget class="QWidget" name="KLADRTreePopup" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="tabPosition" >
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabTree" >
      <attribute name="title" >
       <string>&amp;КЛАДР</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>4</number>
       </property>
       <property name="leftMargin" >
        <number>4</number>
       </property>
       <property name="topMargin" >
        <number>4</number>
       </property>
       <property name="rightMargin" >
        <number>4</number>
       </property>
       <property name="bottomMargin" >
        <number>4</number>
       </property>
       <item>
        <widget class="CKLADRTreeView" name="treeView" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSearch" >
      <attribute name="title" >
       <string>&amp;Поиск</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>4</number>
       </property>
       <property name="topMargin" >
        <number>4</number>
       </property>
       <property name="rightMargin" >
        <number>4</number>
       </property>
       <property name="bottomMargin" >
        <number>4</number>
       </property>
       <property name="horizontalSpacing" >
        <number>4</number>
       </property>
       <property name="verticalSpacing" >
        <number>4</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLineEdit" name="edtWords" />
       </item>
       <item row="0" column="1" >
        <widget class="QToolButton" name="btnSearch" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="CKLADRSearchResult" name="tblSearchResult" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CKLADRTreeView</class>
   <extends>QTreeView</extends>
   <header>KLADRViews.h</header>
  </customwidget>
  <customwidget>
   <class>CKLADRSearchResult</class>
   <extends>QTableView</extends>
   <header>KLADRViews.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
